spec/latest/json-ld-framing/index.html

```diff
@@ -484,8 +474,9 @@
                     as the sole member of a new <em>subjects</em> <a>array</a>,
                     the first value from <em>property</em> in <em>frame</em> as <em>frame</em>,
                     <em>output</em> as <em>parent</em>, and <em>property</em> as <em>active property</em>.</li>
-                  <li>Otherwise, append a copy of <em>item</em> to <a>active property</a> in
-                    <em>output</em>.</li>
+                  <li>Otherwise, if item matches using the <a href="#value-matching">Value matching algorithm</a>,
+                    where the <em>pattern</em> is taken from the first value from <em>property</em> in <em>frame</em>,
+                    append a copy of <em>item</em> to <a>active property</a> in <em>output</em>.</li>
```

Comment:
I think we may need to discuss what's going on here with using the first value as a pattern (to match, I presume in the value matching algorithm), but that can happen post merging this PR. This is at least a start for handling value matching, which we had nothing for previously.